### ISSUE TITLE
module 'meshlabxml' has no attribute 'FilterScript'

### ISSUE THREAD

**Issue by MohamadZohoori** [3DLIRIOUS/MeshLabXML#34]
Hi,

I used to work with meshlabserver in my script and there were no error in code and I was able to get the desired output. But now after 1 year of not using the very same script, I end up facing this error. maybe the function of the library has changed but it used to work with the following script:

scan = mlx.FilterScript(file_in = 'pointcloud.xyz',file_out='%s.stl'%name, ml_version='2016.12')

it is also worth mentioning that I have once uninstalled and reinstalled meshlabxml and probably this new version of installation has caused the error.  Could you please find a way to make it work, or maybe give me the updated script with the same functionality.

**Comment by 3DLIRIOUS** [comment]
Yes, there's been a few changes in meshlabserver, mostly to deal with later versions of MeshLab. Here's some things to check:
- Make sure you are installing meshlabserver from this Github repo. The PyPi version is quite out of date, and PyPi has changed so much that I have to do more work to upload a new version, and i haven't had the time.
- The current recommended version of MeshLab is 2020.09. This is the last version that can run mlx scripts directly (newer versions must invoke mlx scripts using PyMeshLab). Older versions (such as 2016.12) should still work though.

**Comment by MohamadZohoori** [comment]
Many thanks for guiding me through this.
